syntax_and_semantics/type_reflection.md

```diff
@@ -1,3 +1,10 @@
 # Type reflection
 
-Crystal provides basic methods to do type reflection, casting and introspection.
+Crystal provides basic methods to do type reflection, casting and introspection:
+
+* [is_a?](is_a.html)
```

Comment:
While these lists of links are nice, they are going to be hard to maintain and are already available in the sidebar. Unless we can generate this in a more automated way, I would vote against putting redundant lists on the section parent pages.